gita/info.py

```diff
@@ -9,6 +9,35 @@
 from . import common
 
 
+class Truncate():
```

Comment:
This class loads the layout.csv file and is passed to the functions that generate text output. They can then call the truncate function to process their text output.

Reply:
Thank you, I have removed them and pushed the fix.